### PR TITLE
Don't use version namespace for Exception.

### PR DESCRIPTION
**Pull request by zvrba**
The old code will break when Alembic library is upgraded. There's no need to explicitly specify the namespace version (v7) to use `Exception` class because the header defining exception (`Util/Exception.h`) contains code like

```
namespace Alembic { namespace Util { namespace ALEMBIC_VERSION_NS {
class Exception { ... }
}

using namespace ALEMBIC_VERSION_NS;

}}

```

### CODE REVIEW

**Comment by simogasp** [comment]
is it for the latest release of Alembic, 1.6.1? Or the github one?

**Comment by simogasp** [comment]
This PR compiles fine even on older versions of alembic (mine is 1.5.6 on OSX). I guess it was the right thing to do since the beginning :-) And @fabiencastan it may now allow to do what discussed here https://github.com/poparteu/openMVG/pull/236#discussion_r80463204

**Comment by zvrba** [comment]
It is for the latest alembic release (1.6.1).

File: src/software/Localization/convertDescFloat2Uchar.cpp [hunk 1]
```diff
@@ -2,10 +2,13 @@
 #include <openMVG/logger.hpp>
 
 #include <boost/progress.hpp>
-#include <boost/filesystem.hpp>
 #include <boost/program_options.hpp> 
 #include <boost/algorithm/string/case_conv.hpp> 
 
+#define BOOST_NO_CXX11_SCOPED_ENUMS
```
Comment (simogasp) [inline src/software/Localization/convertDescFloat2Uchar.cpp]: u can force push a version without the last commit

Comment (zvrba) [inline src/software/Localization/convertDescFloat2Uchar.cpp]: @fabiencastan Oh, I saw the option, but didn't know what it means. The problem is anyway that I'm using system boost (on Debian) and it's not possible to tell upfront how it's compiled.

Comment (fabiencastan) [inline src/software/Localization/convertDescFloat2Uchar.cpp]: Yes, it's difficult to know how it is compiled in the system packages.
I will try to improve the description of this build option and add a sentence about this in the INSTALL file. And I will merge only the first commit of the PR, if it works for you @zvrba with this cmake build option.

Comment (zvrba) [inline src/software/Localization/convertDescFloat2Uchar.cpp]: I guess it'll work, so just go ahead and merge the first commit.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -427,7 +427,7 @@ ENDIF(OpenMVG_USE_OPENCV)
 # - optional, it allows to compile the vocabulary tree
 # ==============================================================================
 IF(OpenMVG_USE_BOOST)
-  OPTION(BOOST_NO_CXX11 "If Boost is not build with C++11 support this must be enabled to avoid symbol conflicts (SCOPED_ENUM)" OFF)
+  OPTION(BOOST_NO_CXX11 "If Boost is compiled without C++11 support (as it's often the case in OS packages) this must be enabled to avoid symbol conflicts (SCOPED_ENUM)." OFF)
```
Comment (simogasp) [inline src/CMakeLists.txt]: it is often

File: BUILD [hunk 3]
```diff
@@ -24,12 +24,26 @@ As openMVG use some C++11 features you must have a c++11 ready compiler:
...
-OpenMVG_BUILD_TESTS (ON/OFF(default))=> Build openMVG unit tests
-OpenMVG_BUILD_EXAMPLES (ON/OFF(default))=> Build OpenMVG example applications.
-  Does not affect binaries under 'software'
+
+OpenMVG_USE_BOOST (default ON): Use Boost library (enable modules like localization/voctree and other features and optimizations.
+BOOST_NO_CXX11 (default OFF): If your Boost binaries are compiled without C++11 support, you need to set this option to avoid compilation errors.
+                              This is most likely to be the case if you use the system packages to install boost.
+OpenMVG_USE_OPENMP (default ON): Use OpenMP parallelization (huge impact on performances)
+OpenMVG_USE_CCTAG (default ON): Build with CCTag markers support.
+OpenMVG_USE_OPENGV (default OFF): Build with openGV for multi-cameras localization.
```
Comment (simogasp) [inline BUILD]: same here, we can add that it then requires to define a `-DOPENGV_DIR=opengv/install/prefix` at cmake command line.

File: BUILD [hunk 4]
```diff
@@ -24,12 +24,26 @@ As openMVG use some C++11 features you must have a c++11 ready compiler:
...
 --------------------------
-OpenMVG_BUILD_TESTS (ON/OFF(default))=> Build openMVG unit tests
-OpenMVG_BUILD_EXAMPLES (ON/OFF(default))=> Build OpenMVG example applications.
-  Does not affect binaries under 'software'
+
+OpenMVG_USE_BOOST (default ON): Use Boost library (enable modules like localization/voctree and other features and optimizations.
+BOOST_NO_CXX11 (default OFF): If your Boost binaries are compiled without C++11 support, you need to set this option to avoid compilation errors.
+                              This is most likely to be the case if you use the system packages to install boost.
+OpenMVG_USE_OPENMP (default ON): Use OpenMP parallelization (huge impact on performances)
+OpenMVG_USE_CCTAG (default ON): Build with CCTag markers support.
```
Comment (simogasp) [inline BUILD]: we can add that it then requires to define a `-DCCTAG_DIR=cctag/install/prefix` at cmake command line.

File: BUILD [hunk 5]
```diff
@@ -24,12 +24,26 @@ As openMVG use some C++11 features you must have a c++11 ready compiler:
...
+OpenMVG_USE_OPENGV (default OFF): Build with openGV for multi-cameras localization.
+OpenMVG_USE_ALEMBIC (default OFF): Build with Alembic file format support
```
Comment (simogasp) [inline BUILD]: this is tougher as it requires several definition
```
ALEMBIC_ROOT
ALEMBIC_HDF5_ROOT
ALEMBIC_ILMBASE_ROOT
ALEMBIC_OPENEXR_ROOT
```

On the other hand, if we switch to the latest version of alembic we can get rid of our FindAlembic and use the AlembicConfig.cmake that is generated with all the needed dependencies. In this case we just need to define `-DAlembic_DIR=/alembici/install/prefix/lib/cmake/Alembic/`  (where AlembicConfig.cmake can be found)

File: BUILD [hunk 6]
```diff
@@ -24,12 +24,26 @@ As openMVG use some C++11 features you must have a c++11 ready compiler:
...
+OpenMVG_USE_OPENCV (default: OFF): Build with openCV
```
Comment (simogasp) [inline BUILD]: `-DOpenCV_DIR=/opencv/install/prefix/share/OpenCV/` where OpenCVConfig.cmake can be found)

File: BUILD (hunk 3 above)
Comment (simogasp) [inline BUILD]: also for now it works better with the fork in https://github.com/poparteu/opengv until PRs will be merged...

File: INSTALL.md [hunk 7]
```diff
@@ -27,22 +27,53 @@ As openMVG use some C++11 features you must have a c++11 ready compiler:
...
+Build OpenMVG documentation
+
+* OpenMVG_BUILD_EXAMPLES (default ON)
+Build OpenMVG samples applications (openMVG softwares are still builded)
+
+* OpenMVG_BUILD_OPENGL_EXAMPLES (default OFF)
+Build OpenMVG openGL examples
+
+* OpenMVG_BUILD_COVERAGE (default OFF)
+Enable code coverage generation (gcc only)
```
Comment (simogasp) [inline INSTALL.md]: while you are at it, you can correct L:80  'General informations or openMVG SfM pipelines' as it does not render well on markdown if splitted into two lines

**Pull request merged**